charts/spire/charts/spiffe-oidc-discovery-provider/templates/tests/test-connection.yaml

```diff
@@ -19,13 +20,13 @@ spec:
     - name: wget-service-name-namespace
       image: busybox
       command: ['wget']
-      args: ['-O', '/dev/null', '{{ include "spiffe-oidc-discovery-provider.fullname" . }}.{{ .Release.Namespace }}:{{ .Values.service.port }}/.well-known/openid-configuration']
+      args: ['-O', '/dev/null', '{{ include "spiffe-oidc-discovery-provider.fullname" . }}.{{ include "spiffe-oidc-discovery-provider.namespace" . }}:{{ .Values.service.port }}/.well-known/openid-configuration']
```

Comment:
Why hiding test output? Makes it harder to debug.

Reply:
That part is from the other review. The only thing this does is change the namespace variable.